### PR TITLE
Adds filter addresses to tokens endpoint

### LINKED ISSUE #921: Add filter in GET /tokens to get by SC addr

### PR DESCRIPTION
**Pull request by ARR552**
Closes #921

### What does this PR does?

This PR allows filter tokens using addresses.
Also, in this PR has change comma by pipeline to be mor consistent with api rest.

Example:
http://<ip_addr>:<port>/v1/tokens?addresses=0xaded47e7b9275b17189f0b17bf6a4ce909047084|0xe07d8aa8b0d6b390fbde895271601bc27ba89fe2

### How to test?

Run the hermez node and call the api.

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [X] Include tests
- [X] Respect code style and lint
- [X] Update swagger file (if needed)
- [ ] Update documentation (*.md) (if needed)

### CODE REVIEW

File: api/parsers/tokens.go [hunk 1]
```diff
@@ -55,15 +57,25 @@ func ParseTokensFilters(c *gin.Context) (historydb.GetTokensAPIRequest, error) {
 
 	var symbols []string
 	if tokensFilters.Symbols != "" {
-		symbols = strings.Split(tokensFilters.Symbols, ",")
+		symbols = strings.Split(tokensFilters.Symbols, "|")
+	}
+
+	var tokensAddress []ethCommon.Address
```
Comment (Mikelle) [inline api/parsers/tokens.go:63]:
```suggestion
	var tokensAddresses []ethCommon.Address
```

**Review by Mikelle — APPROVED** [review]

File: api/swagger.yml [hunk 2]
```diff
@@ -1264,22 +1264,30 @@ paths:
...
             example: "2,44,689"
         - name: symbols
           in: query
           required: false
           description: Include only specific tokens by their symbols.
           schema:
             type: string
-            description: Comma separated list of token symbols.
+            description: Pipeline "|" separated list of token symbols.
             example: "DAI,NEC,UMA"
```
Comment (marioidival) [inline api/swagger.yml:1276]:
```suggestion
            example: "DAI|NEC|UMA"
```

File: api/swagger.yml [hunk 3]
```diff
@@ -1264,22 +1264,30 @@ paths:
           description: Include only specific tokens by their Hermez identifiers.
           schema:
             type: string
-            description: Comma separated list of token identifiers
+            description: Pipeline "|" separated list of token identifiers
             example: "2,44,689"
```
Comment (marioidival) [inline api/swagger.yml:1268]:
```suggestion
            example: "2|44|689"
```

**Review by marioidival — CHANGES_REQUESTED** [review]
Missing update others examples with |

**Review by marioidival — APPROVED** [review]

**Pull request merged**